event_server.cpp

```diff
@@ -1861,6 +1861,36 @@ static void get_settling(JObj& response, const json_value *params)
     response << jrpc_result(settling);
 }
 
+static void get_variable_delay_settings(JObj& response, const json_value* params)
+{
+    JObj rslt;
+
+    VarDelayCfg delayParams = pFrame->GetVariableDelayConfig();
+    rslt << NV("Enabled", delayParams.enabled)
+        << NV("ShortDelay", delayParams.shortDelay / 1000)
+        << NV("LongDelay", delayParams.longDelay / 1000);
+    response << jrpc_result(rslt);
+}
+
+static void set_variable_delay_settings(JObj& response, const json_value* params)
+{
+    Params p("Enabled", "ShortDelay", "LongDelay", params);
+    const json_value* p0 = p.param("Enabled");
+    const json_value* p1 = p.param("ShortDelay");
+    const json_value* p2 = p.param("LongDelay");
+    bool enabled;
+    double shortDelay;
+    double longDelay;
+    if (!p0 || !p1 || !p2 || !bool_param(p0, &enabled) || !float_param(p1, &shortDelay) || !float_param(p2, &longDelay))
+    {
+        response << jrpc_error(JSONRPC_INVALID_PARAMS, "expected Enabled, ShortDelay, LongDelay params)");
+        return;
+    }
+    VarDelayCfg currParams;
+    pFrame->SetVariableDelayConfig(enabled, (int)(shortDelay * 1000.), (int)(longDelay * 1000.));

```

Comment:
now that I understand these delays are always positive integer number of seconds, I think we should change this back to the way you had it before since this way would now allow values that are millisecond resolution and we do not want to allow that.
I would suggest adding "Seconds" to the API parameter names and variable names so that it is clearer that we're dealing with seconds (and not milliseconds like all other durations in the code base)
```suggestion
    pFrame->SetVariableDelayConfig(enabled, (int) shortDelaySeconds * 1000), (int) longDelaySeconds * 1000));
```